.nvmrc

```diff
@@ -1 +1 @@
-16.13.2
+18.13.0
```

Comment:
Sorry az teraz som si vsimol ze [v README je spomenuta este neaktualna node verzia](https://github.com/slovensko-digital/priznanie-digital#how-to-develop), ak to tam mozes prosim tiez updatnut.

Reply:
skor by som to vratil lebo je to jedno a ide to na oboch, a dajme to zvlast a niekto by mal dohliadnut ze kde vsade to treba riesit a riesme to v separatnom PR